scripts/verify-r1-native-isolation.ps1

```
@@ -0,0 +1,2133 @@
+    $records = New-Object System.Collections.ArrayList
+    foreach ($case in $cases) {
+        [void] (Stop-IsolatedDaemon -Reason ("admission-pre-{0}" -f $case.name))
+        $captureRelative = "artifacts/captures/admission-$($case.name).ndjson"
+        $capturePath = Join-Path -Path $script:OutputDirFull -ChildPath $captureRelative
+        $arguments = @($script:Policy, $script:ModernFixture)
+        $run = Invoke-NativeProcess -FilePath $script:MuxBinary -Arguments $arguments -WorkingDirectory $script:SourceRootFull -StdinText ($case.raw + "`n") -EnvironmentOverrides (Get-ProductEnvironment -CapturePath $capturePath) -TimeoutSeconds 20
+        Assert-Verification ($run.exit_code -ne 0) "admission case $($case.name) unexpectedly succeeded"
+        $messages = @(ConvertFrom-NdjsonText -Text $run.stdout -Label "admission case $($case.name)")
+        Assert-Verification ($messages.Count -eq 1) "admission case $($case.name) emitted $($messages.Count) stdout frame(s), want one error"
+        $message = $messages[0].message
+        $error = Get-JsonProperty -Object $message -Name "error"
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'error' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)